spec/07-variables.md

```diff
@@ -259,6 +281,8 @@ function f()
 }
 ```
 
+Be also aware that declaring a variable global can hide a local variable and/or a function static with the same name. See ([§§](#hidingNotice)) for an example.
```

Comment:
I'd rewrite it as "local or static variable with the same name". Also, we try to get rid of [§§] stuff. Note that manual doesn't have them anymore in current branches.